server.js

```diff
@@ -0,0 +1,13 @@
+const express = require("express");
+const app = express();
+const port = 3000;
+
+app.route("/ping").get((req, res) => {
```

Comment:
Consider adding error handling to the endpoint. This can be done by adding a middleware function that catches any errors and sends a response with the error message and status code. [important]